R/utilities-units.R

```diff
@@ -74,3 +81,15 @@ toDisplayUnit <- function(quantity, values) {
   validateIsOfType(quantity, Quantity)
   toUnit(quantity, values, quantity$displayUnit)
 }
+
+
+#' Returns the name of all available dimensions defined in the OSPSuite platform
+#'
+#' @examples
+#' dims <- allAvailableDimensions()
+#'
+#' @export
+allAvailableDimensions <- function(){

```

Comment:
@pchelle This also returns all dimensions available in the system. You need to use one of those names when you do conversion if the dimension name is not avialable from the quantity